packages/core/src/components/utils/input/index.ts

```diff
@@ -93,7 +93,7 @@ export interface IxInputFieldComponent<T = string>
   getNativeInputElement(): Promise<HTMLInputElement | HTMLTextAreaElement>;
 
   // Annotate as @Method()
-  focusInput(): void;
+  focusInput(): void | Promise<void>;
```

Comment:
```suggestion
  focusInput(): Promise<void>;
```
We can retype it as `Promise` only because annotation with `@Method` expects always a promise as return value.
https://stenciljs.com/docs/methods#public-methods-must-be-async